src/index.js

```diff
@@ -0,0 +1,132 @@
+const state = {
+  temp: 68
+};
+
+let isFahrenheit = true;
+
+const vibe = {
+  sheet: 2
+}
+
+let isReset = false;
```

Comment:
If using a grouping object like `state`, prefer to put more of the values within the object. It's fine to forgo using a grouping object as well, and storing all these values in individual variables as well.